src/zc/table/table.py

```diff
@@ -213,7 +218,7 @@ def __getitem__(self, key):
                 return items.__getitem__(key)
             except (AttributeError, TypeError):
                 if stride != 1:
-                    raise NotImplemented
+                    raise TypeError()
```

Comment:
NotImplementedError() would be a better choice

Reply:
done